lesson_26/api/java/api_app/src/main/java/com/codedifferently/lesson26/web/MediaItemsController.java

```diff
@@ -22,11 +33,80 @@ public MediaItemsController(Library library) throws IOException {
     this.librarian = library.getLibrarians().stream().findFirst().orElseThrow();
   }
 
-  @GetMapping("/items")
-  public GetMediaItemsResponse getItems() {
+  @GetMapping()
+  public ResponseEntity<GetMediaItemsResponse> getItemById() {
     Set<MediaItem> items = library.search(SearchCriteria.builder().build());
+    if (items.isEmpty()) {
+      ResponseEntity.noContent();
```

Comment:
Nope, should return `GetMediaItmesResponse` with an empty list of items. Anything else would be unexpected and unusual.